tests/mocha/navigation_test.js

```diff
@@ -209,17 +209,6 @@ suite('Navigation', function() {
           Blockly.Navigation.STATE_WS);
     });
 
-    test('Add block to workspace with a marked workspace node', function() {
```

Comment:
Why did this test break?

Reply:
We removed the function that this test was testing a while ago and I think we just never removed this test.